src/olympia/versions/models.py

```diff
@@ -549,7 +549,7 @@ def reset_nomination_time(self, nomination=None):
 
     @property
     def is_listed(self):
-        return self.addon.is_listed
+        return self.channel == amo.RELEASE_CHANNEL_LISTED
 
```

Comment:
We should either drop this method entirely now, or leave it as is until clean up.  Any code calling it is not channel aware.